### DISCUSSION TITLE
For what can I use the "PriceCategories"? How to do something like "StrikeThrough Prices"?

### DISCUSSION THREAD

**Discussion by vorlontech** [OFFLINE-GmbH/oc-mall-plugin#1165] [Q&A]
hi,
I don't get the idea behind the "PriceCategories"? For what can I use them?
Have added one PriceCategory and now I have the problem, that I can't find how to access this price. Can someone let me know?
Is this the way to build up something like "StrikeThrough Prices"? My client needs to have them.
Can this be done in current version and how?
Thanks very much
Vorlon.Tech

**Comment by tobias-kuendig** [comment]
The price categories allow you to specify any custom additional price information for a product. It can be purely informational.

Your "strike through prices" requirement sounds like a good use-case. You can define an "old price" (199 USD) and an effective price (99 USD) which will be used for the product during checkout. 

If you define an "old price" category (with the slug `old_price`) for this, you can access the information like so out-of-the-box:
https://github.com/OFFLINE-GmbH/oc-mall-plugin/blob/70974d90523e7975731e629dfb7303f9f1f356de/components/product/price.htm#L8
